### PR TITLE
Recognize more breach  location types

### PR DESCRIPTION
**Pull request by SiggyF**
Implements  https://issuetracker.deltares.nl/browse/LIWO-482

### CODE REVIEW

File: src/lib/leaflet-utils/markers.js [hunk 1]
```diff
@@ -10,6 +10,20 @@ import iconShadowUrl from '../../img/markers/marker-shadow.png'
 import redIconUrl from '../../img/markers/marker-icon-red.png'
 import redRetinaIconUrl from '../../img/markers/marker-icon-2x-red.png'
 
+// disable camelcase warning because we want to separate numbers
```
Comment (CindyvdVries) [inline src/lib/leaflet-utils/markers.js:13]: A bit ugly bypassing the linter in this way. Is the 60_60 needed everywhere? Why not make is icon30Url etc?

Comment (SiggyF) [reply src/lib/leaflet-utils/markers.js:13]: 30_60_60 refers to the  HSL values.  It's a [common](https://colorcet.holoviz.org/) way to refer to color styles.

Comment (SiggyF) [reply src/lib/leaflet-utils/markers.js:13]: I documented this in the code.

**Pull request merged**